.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
-# Description
+## Description
 
 Fixes #issue.
 
-# Test Plan
+## Test Plan
 
+- [ ] Confirm that the CI build is successful
 
-# Checklist
+## How to create a useful test plan
 
-If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:
+### Qubes OS environment
 
- - [ ] I have tested these changes in the appropriate Qubes environment
- - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
- - [ ] These changes should not need testing in Qubes
+If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes OS testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes OS is required. Please make sure you mention it in the **test plan**.
```

Comment:
nit: "network (via the RPC service) traffic" -> "network traffic (via the RPC service)"